cdap-data-fabric/src/main/java/co/cask/cdap/data2/metadata/dataset/MetadataDataset.java

```diff
@@ -53,12 +54,12 @@
     .registerTypeAdapter(Id.NamespacedId.class, new NamespacedIdCodec())
     .create();
 
-  private static final String HISTORY_COLUMN = "h";
-  private static final String VALUE_COLUMN = "v";
-  private static final String TAGS_SEPARATOR = ",";
+  private static final Pattern VALUE_SPLIT_PATTERN = Pattern.compile("\\s*[-_,\\s]\\s*");
 
-  static final String KEYVALUE_COLUMN = "kv";
-  static final String CASE_INSENSITIVE_VALUE_COLUMN = "civ";
+  private static final String HISTORY_COLUMN = "h"; // column for metadata history
+  private static final String VALUE_COLUMN = "vc";  // column for metadata value
+  static final String INDEX_COLUMN = "ic";          // column for metadata indexes
```

Comment:
nit: better to have this after the next constant to keep all the private members together?